src/responder/multiResponder.ts

```diff
@@ -207,4 +213,10 @@ export class MultiResponder {
             logger.error(doh);
         }
     }
+
+    public async checkBalance(){
+        if((await this.signer.provider!.getBalance(this.address)).gt(this.balanceTreshold)){
+            logger.error("Responder balance is becoming low. Current balance: "+ (await this.signer.provider!.getBalance(this.address)));
```

Comment:
Calling getBalance twice means you make two network requests. Better to assign the result to a variable instead.